spec/requests/posts_spec.rb

```diff
@@ -0,0 +1,30 @@
+require 'rails_helper'
+
+RSpec.describe 'users', type: :request do
+  describe 'GET/index' do
+    before :each do
+      get '/users/:user_id/posts/'
+    end
+    it 'is a success' do
+      expect(response).to have_http_status(:success)
+    end
+    it 'should render index' do
+      expect(response).to render_template(:index)
+    end
+    it 'Should return the correct placeholder' do
+      expect(response.body).to include('Posts')
+    end
+  end
+
+  describe 'GET/show' do
+    before :each do
+      get '/users/:user_id/posts/'
```

Comment:
- [ ] Same here, Kindly get the correct route for the post-show controller which is ` /users/:user_id/posts/:id`.